docs/src/pages/components/autocomplete/FreeSoloCreateOption.tsx

```diff
@@ -11,19 +11,28 @@ export default function FreeSoloCreateOption() {
   return (
     <Autocomplete
       value={value}
-      onChange={(event: any, newValue: FilmOptionType | null) => {
+      onChange={(event, newValue) => {
+        if (newValue === null) {
+          setValue(newValue);
+          return;
```

Comment:
use if-else instead of early return. Makes it easier to follow the control-flow.

Reply:
I got this habit from https://github.com/rubocop-hq/ruby-style-guide#no-nested-conditionals. if/else sound great as long as the conditional logic is flat. If it's nested, I'm not so sure 🤔.

Reply:
Linked article is about ruby and nested conditionals neither of which apply here.